library/ssl_tls.c

```diff
@@ -2822,54 +2881,168 @@ static void ssl_swap_epochs( mbedtls_ssl_context *ssl )
 
 /*
  * Retransmit the current flight of messages.
+ */
+int mbedtls_ssl_resend( mbedtls_ssl_context *ssl )
+{
+    int ret = 0;
+
+    MBEDTLS_SSL_DEBUG_MSG( 2, ( "=> mbedtls_ssl_resend" ) );
+
+    ret = mbedtls_ssl_flight_transmit( ssl );
```

Comment:
This line could be moved to the int ret = [...] line, but I guess that if you want to have debugs in the right place... cosmetic remark.

Reply:
I prefer it this way, leaving the debugging message signalling the function entry prior to what the function actually does.